### PR TITLE
feat(home): add Wallora logo to header

### PR DESCRIPTION
**Pull request by priyankajayaswal**
Fixes: #54 
with material-ui logout case
image

with material-ui login case
image

Footer
image

### CODE REVIEW

File: components/Footer.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export default function Footer(){
     return(
-        <div className="bg-blue-200 p-5 mt-2 flex justify-between">
+        <div className="bg-violet-200 p-5 mt-2 flex justify-between">
```
Comment (souvikbasu) [inline components/Footer.tsx:3]: should be using theme variable instead like `primary-color`

File: components/Header.tsx [hunk 2]
```diff
@@ -1,9 +1,11 @@
 export default function Header(props:any){
+    const className = "p-5 prominent-button"
     return(
-        <div className="bg-green-200 pl-10 pt-5 pb-5 flex justify-between">
+        <div className="bg-violet-200 pl-10 pt-5 pb-5 flex justify-between">
+            <img src="./images/wallora-logo-170x53.jpeg"></img>
```
Comment (souvikbasu) [inline components/Header.tsx:5]: add alt text for image

File: components/Header.tsx [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
             <span>This is header</span>
             {
-                props.user ? <a href="/api/auth/logout" className='pr-10'>LOGOUT</a> : <a href="/api/auth/login" className='pr-10'>LOGIN</a> 
+                props.user ? <a href="/api/auth/logout" className={className}>Logout</a> : <a href="/api/auth/login" className={className}>Login</a>
```
Comment (souvikbasu) [inline components/Header.tsx:8]: use tailwind classes instead of className

File: pages/_app.tsx [hunk 4]
```diff
@@ -4,6 +4,7 @@ import { UserProvider } from '@auth0/nextjs-auth0';
 import { useEffect } from 'react'
 import { useRouter } from 'next/router'
 
+import "./styles/app.css"
```
Comment (souvikbasu) [inline pages/_app.tsx:7]: do not use custom CSS if can be done using tailwind

File: pages/styles/app.css [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.prominent-button {
```
Comment (souvikbasu) [inline pages/styles/app.css:1]: integrate material UI lib

File: tailwind.config.js [hunk 6]
```diff
@@ -7,6 +7,11 @@ module.exports = {
         'sprite-background': "url('/images/icons.png')",
       })
     },
+    colors: {
+      violet: {
+        200: "#f4dcf4"
```
Comment (souvikbasu) [inline tailwind.config.js:12]: set theme variable name as `primary-color`

Comment (souvikbasu) [reply tailwind.config.js:12]: so that tomorrow we can decide to use another color without having to change its usage at all places

**Review by souvikbasu — CHANGES_REQUESTED** [review]
since Aryaman was assigned to the issue, let's allow him to send the PR if he can

**Comment by aryamantodkar** [comment]
> since Aryaman was assigned to the issue, let's allow him to send the PR if he can

I have assigned myself to the terms and privacy issue

**Review by souvikbasu — APPROVED** [review]

**Pull request merged**